samples/oauth_react/src/Profile.jsx

```diff
@@ -13,41 +13,30 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { withAuth } from '@okta/okta-react';
 import { Header, Icon, Table } from 'semantic-ui-react';
+import { SignedIn } from './Authentication';
 
-import { checkAuthentication } from './helpers';
+class Profile extends Component {
+  static propTypes = {
+    user: PropTypes.object
+  };
 
-export default withAuth(class Profile extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { userinfo: null, ready: false };
-    this.checkAuthentication = checkAuthentication.bind(this);
+  renderEmptyState() {
+    return <div><p>Your user info is unavailable.</p></div>;
   }
 
-  async componentDidMount() {
-    await this.checkAuthentication();
-    this.applyClaims();
-  }
+  renderProfileDetails() {
+    console.log(this.props.user);
 
-  async componentDidUpdate() {
-    await this.checkAuthentication();
-    this.applyClaims();
-  }
+    const Claim = ({claimName, claimValue}) => {
+      const claimId = `claim-${claimName}`;
+      return <tr key={claimName}><td>{claimName}</td><td id={claimId}>{claimValue}</td></tr>;
+    };
 
-  async applyClaims() {
-    if (this.state.userinfo && !this.state.claims) {
-      const claims = Object.entries(this.state.userinfo);
-      this.setState({ claims, ready: true });
-    }
-  }
-
-  render() {
     return (
       <div>
```

Comment:
Think you can remove this `<div>`, only wraps a single `<div>` with the code removal on lines 49, 50